### ISSUE TITLE
[BUG] Run with CLI

### ISSUE THREAD

**Issue by BlackYuzia** [javascript-obfuscator/javascript-obfuscator#904]
### Example

```bash
javascript-obfuscator index.js --config config.json # work
```

```bash
javascript-obfuscator --config config.json  index.js # error
# return: ReferenceError: Given input path must be a valid `.js` file or directory path
```

### Expects

file what would be obfuscated should be parse fine. Flags, like `--config` or any else (like `-o` or `--output` and etc) shouldn't destroy command run if flags would be placed before js file.

**Comment by sanex3339** [comment]
Hi. Hm. Currently i got the file path here:
https://github.com/javascript-obfuscator/javascript-obfuscator/blob/master/src/cli/JavaScriptObfuscatorCLI.ts#L140

Maybe I can change the logic to parse arguments and detect argument with the input file path

**Comment by sanex3339** [comment]
Will fix it today. PR is almost ready.

**Comment by sanex3339** [comment]
Released as 2.11.1
